xarray/backends/api.py

```diff
@@ -825,6 +826,10 @@ def open_mfdataset(
         - 'override': if indexes are of same size, rewrite indexes to be
           those of the first object with that dimension. Indexes for the same
           dimension must have the same size in all objects.
+    master_file : int or str, optional
```

Comment:
This is netcDF4's documentation for `master_file`:

>  file to use as "master file", defining all the variables with an aggregation dimension and all global attributes.

let's make it clear that unlike netCDF4 we are only using this for attributes

Reply:
Do you suggest to use a different keyword, maybe `attrs_file`?
Or just clarify the difference in the docs? I don't mind.
@dcherian Thanks for the review!

Reply:
I was initially thinking of just adding a line to the docstring but we should think about renaming this to something like `attrs_from`?

Reply:
So I've renamed it to `attrs_file` to avoid confusion with netCDF4. Thanks for pointing that. I am open to any name as long as the option is here.

Reply:
@dcherian can we mark this as resolved? The `attrs_file` now only accept a file name (see other conversation below).